packages/ui/exports/core.js

```diff
@@ -1,7 +1,7 @@
 export { DisabledMixin } from '../components/core/src/DisabledMixin.js';
 export { DisabledWithTabIndexMixin } from '../components/core/src/DisabledWithTabIndexMixin.js';
 export { ScopedStylesController } from '../components/core/src/ScopedStylesController.js';
-export { SlotMixin } from '../components/core/src/SlotMixin.js';
+export { SlotMixin, moveDefaultSlottablesToTarget } from '../components/core/src/SlotMixin.js';
```

Comment:
This suggests it is a universal utility that can be used in any context y all Lion consumers. However, it only works with SlotMixin. 
Exposing it means increasing the maintenance/support surface, where we would rather keep its exposure limited (also since the need for it will disappear once cross root aria is available: https://lion-web.netlify.app/_merged_assets/_static/crossRootAriaLion.pdf)